src/dune_rules/alias0.ml

```diff
@@ -14,5 +14,6 @@ let doc_json = standard "doc-json"
 let doc_new = standard "doc-new"
 let check = standard "check"
 let install = standard "install"
+let ocaml_index = standard "ocaml-index"
```

Comment:
if we end up making a new standard index, it'll need to be documented in `doc/reference/aliases.rst`

Reply:
todo @voodoos: I will add documentation when we will have settled on the additional alias(es) we want.

Reply:
done in [4d13b4d](https://github.com/ocaml/dune/pull/10422/commits/4d13b4d1d00e3f49166bddc0e5a94d52c68a1044)